lib/redux/actions/async.js

```diff
@@ -422,105 +422,134 @@ export function doVersionCheck() {
  * COMPLEX ACTION CREATORS
  */
 
-export function putTargetsInStorage() {
+export function setTargetTimezone(userId, timezoneName) {
   return (dispatch, getState) => {
-    const { targetDevices, targetTimezones, uploadTargetUser } = getState();
-    let usersWithTargets = {};
-    _.forOwn(targetDevices, (devicesArray, userId) => {
-      usersWithTargets[userId] = _.map(devicesArray, (deviceKey) => {
-        return {key: deviceKey};
-      });
+    const { api } = services;
+    dispatch(syncActions.updateProfileRequest());
+    api.user.updateProfile(userId, {
+      'patient': {
```

Comment:
Nitpick, but I prefer not to use strings as key values unless there's an illegal character (like '-') involved. So instead of `{'patient': ...}`, `{patient: ...}`. I think we're pretty consistent about this one in the uploader.